Extend badges in readme and add logo

davidrudlstorfer opened the pull request:
<!--
* Title: Provide a general summary of your changes in the Title above.

* Assignees:  If you know anyone who should likely handle bringing this pull request (PR) to completion -- that's usually yourself -- select them from the Assignees drop-down on the right.

* Labels: Update the label of the issue(s) addressed by this pull request to "Under Review".
-->

## Description and Context
<!--
Provide a brief and concise description of your proposed change. Questions you should think about:
* Why is this change required?  What problem does it solve?
* Is there a bigger picture? Is this PR a part of a larger set of changes? Which further steps are planned after merging this PR, if any?
* How has the proposed implementation been verified and tested?

Keep the description of the PR always up-to-date and concise.
-->

Added the current 4C logo as a replacement to the name 4C in the header

Also saved the static badges into assets and added them to the readme

## Related Issues and Merge Requests
<!--
If applicable, let us know how this pull request is related to any other open issues or pull requests by linking to them here.
Some suggestion for keywords:
Closes (will automatically close mentioned issue if merged), Blocks, Related to
-->

#42

davidrudlstorfer commented:
![image](https://github.com/user-attachments/assets/b0b188b9-4c85-4dd9-977a-f7c30e511eaf)

davidrudlstorfer commented:
@amgebauer the github logo is still pulled from github within the SVG file for the badges - we could also remove the Github logo

sebproell commented:
> @amgebauer the github logo is still pulled from github within the SVG file for the badges - we could also remove the Github logo

I think we can just drop the logo here although I guess pulling from Github is OK since we are on Github anyway. The reason why I would drop it is that these badges are not really related to Github. We self-host the website and the docs are separate from the repo as well. Yes, we create them via actions, but the badge is not really about the development on Github which is the case for the workflow badges.

davidrudlstorfer commented:
> > @amgebauer the github logo is still pulled from github within the SVG file for the badges - we could also remove the Github logo
> 
> I think we can just drop the Github logo here although I guess pulling from Github is OK since we are on Github anyway. The reason why I would drop it is that these badges are not really related to Github. We self-host the website and the docs are separate from the repo as well. Yes, we create them via actions, but the badge is not really about the development on Github which is the case for the workflow badges.

Done ✅

amgebauer commented on assets/logo/4c-logo.svg:
I'm not sure whether we want to add this logo now. We never agreed on a logo yet. Maybe just leave the 4C title for now.

davidrudlstorfer commented:
@sebproell I've moved the `assets` into `utilities`

@amgebauer as discussed the logo is identical to the website, once a new logo is designed both need to be updated

mayrmt commented:
I appreciate the initiative, but there is no official logo yet, so please do not put one into the repository. Thanks!

mayrmt reviewed (changes requested):
I appreciate the initiative, but there is no official logo yet, so please do not put one into the repository. Thanks!

(Yes, there's a logo on the website. In my opinion, it should not be there either.)

davidrudlstorfer commented:
@mayrmt thanks for your feedback 🙏🏻

I've removed the logo from the readme as well as from the utilities and adjusted the commit message accordingly.

To stay consistent between the Github repo and our website we should also remove the logo from the website ASAP and create an issue that we need a logo for the usage within the repo and on the website. Maybe someone can remove the logo from the website

Therefore, this MR would be ready for review again
